k8s-readonly.tf

```diff
@@ -171,6 +171,14 @@ resource "kubernetes_deployment" "apiary_hms_readonly" {
             name  = "HMS_MAX_THREADS"
             value = local.hms_ro_maxthreads
           }
+          env {
+            name  = "MYSQL_CONNECTION_POOL_SIZE"
+            value = var.hms_ro_db_connection_pool_size
+          }
+          env {
+            name  = "HMS_AUTOGATHER_STATS"
+            value = "false"
```

Comment:
I set this explicitly to `false` on the RO HMS instance because I wasn't sure at what point a `create table` will fail on the RO HMS.  If it happens after stats gathering, then the default of `hive.stats.autogather=true` would just cause us to waste time and block threads.

I'm happy to be talked out of it, though.